### ISSUE TITLE
Errors building for Houdini 20.5.278 (Py3.11) on Windows

### ISSUE THREAD

**Issue by dovanbel** [LucaScheller/VFX-UsdAssetResolver#13]
Hi,

I'm trying to build the cached resolver using the batch file on Windows.

In the bat file, I specified the correct path for Houdini 20.5.278. I also changed this line 
`cmake . -B build -G "Visual Studio 16 2019" -A x64 -T v142`
to
`cmake . -B build -G "Visual Studio 17 2022" -A x64`
Because the 20.5.278 version is built with v143 tools

I had other errors, so I also changed the contexts of the CMakeLists.txt file, changing every occurrence of Python3.10 to Python3.11, as this is the version of Python used in Houdini 20.5.278

I still get errors when launching the build, I'm attaching the error log
[error.log](https://github.com/user-attachments/files/16382795/error.log)

Unfortunately I don't know C++, so I hope @LucaScheller that you will have some time to look into this :-)

Regards

**Comment by LucaScheller** [comment]
Hey,
the error came from the c++ standard not being set in the CMake file.
I've updated it (by setting `set(CMAKE_CXX_STANDARD 17)`) and by also updating the automatic build configuration to run on VS 2022.

So now we have automatic builds for H20.5 on Windows and Linux again, see [Release v0.7.1](https://github.com/LucaScheller/VFX-UsdAssetResolver/releases/tag/v0.7.1) :) 
Let me know if that fixes your problem (I didn't test the actual build result on Windows)!
Cheers,
Luca

**Comment by dovanbel** [comment]
Hi @LucaScheller,

Thanks a zillion times 👍 

I tried building 0.7.1 on my windows machine and got no errors. I downloaded the binaries and started testing it. It loads in Houdini 20.5.278, I can see that the default resolver is being set to the CachedResolver. I have not tested yet if it actually works (have some unrelated bugs in the new version of Arnold to solve first). I'll report back if there are any issues.

Thanks again
